### PR TITLE
[13.0][MIG] product_multi_category

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

**Comment by okuryan** [comment]
@pedrobaeza Is it merge request ready to be merged? Looks like all checks passed

**Comment by pedrobaeza** [comment]
It requires 2 reviews to be merged.

**Comment by okuryan** [comment]
@pedrobaeza who can be a reviewer? Anybody?

**Comment by pedrobaeza** [comment]
One of them can be. The other should be someone in the repository PSC. See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#review

**Comment by okuryan** [comment]
@pedrobaeza who is PSC here?

**Comment by pedrobaeza** [comment]
Check it in https://odoo-community.org/psc-teams

File: product_multi_category/models/product.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2009 Akretion,Guewen Baconnier,Camptocamp,Avanzosc,Sharoon Thomas,Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    categ_ids = fields.Many2many(
```
Comment (emagdalenaC2i) [inline product_multi_category/models/product.py:10]: Why do you remove categ_id = fields.Many2one(string='Pricing/Primary Category') ?

Comment (emagdalenaC2i) [reply product_multi_category/models/product.py:10]: Ok, I tested it and seems that it is not necessary

**Review by emagdalenaC2i — APPROVED** [review]

**Review by ventor-dev — APPROVED** [review]
I have also reviewed this module and approved it.

Note that in general idea of this module is quite questionable. It looks like "mixing apples and oranges". Both of them are fruits... but quite different still. For eCommerce integration for categories it is better to create new "ecommerce.product.category" that will be simple model without all the functionality related to accounting.

"product.category" object should not be touched at all here. As we are misusing it.

But as this approach exists already for years, have no choice just to approve it. So work on connectors can continue.

**Comment by okuryan** [comment]
@lasley can you merge this?

**Comment by mlaitinen** [comment]
Merge failed, here's the failed check: https://travis-ci.org/github/OCA/product-attribute/jobs/656130694?utm_medium=notification&utm_source=github_status

Looks like the problem isn't in this module, but in `product_restricted_type`.

**Comment by ps-tubtim** [comment]
@mlaitinen I try to rebase. Please merge again.

**Pull request merged**